t/batches.t

```diff
@@ -2,6 +2,7 @@ use strict;
 use warnings;
 use 5.14.2;
 
+use Capture::Tiny qw( capture );
```

Comment:
This is a new dependency. Maybe add it as `test_requires` in Makefile.PL and the installation instruction too?

Reply:
Not applicable anymore since we now rely on a DBI attribute as you suggested.